#!/usr/bin/env python
PACKAGE = "grid_map_2d_mapper"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_height", double_t,    0, "Min Height"  , 0.25,  -100.0, 100.0)
gen.add("max_height", double_t,    0, "P Max Height", 1.00,  -100.0, 100.0)
gen.add("mapping_active", bool_t,    0, "Mapping active"  , True)
gen.add("min_floor_height", double_t,    0, "Min Floor Height"  , -0.15,  -10.0, 10.0)
gen.add("max_floor_height", double_t,    0, "Max Floor Height"  , 0.15,  -10.0, 10.0)

exit(gen.generate(PACKAGE, "grid_map_2d_mapper", "GridMap2DMapper"))
